### PR TITLE
Add TS types definitions

### PR DESCRIPTION
**Pull request by toomuchdesign**
### What kind of change does this PR introduce? _(Bug fix, feature, docs update, ...)_

Feature

### What is the new behaviour?

TS definitions support

### Does this PR introduce a breaking change? _(What changes might users need to make in their application due to this PR?)_

Yes for TS users

### Please check if the PR fulfills these requirements:

- [x] Tests for the changes have been added
- [x] Docs have been added / updated

### CODE REVIEW

**Comment by toomuchdesign** [comment]
Hi @manosim,
here is the PR related to TS types definitions for react-minimal-pie-chart. I'd be glad if you could take a look at it. :)

Greetings!

File: src/types/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+
+declare type StyleObject = {
+  [key: string]: string | number;
+};
+
+export type DataEntry = {
```
Comment (manosim) [inline src/types/index.d.ts]: Not an issue - just my opinion - should the name of this be more specific to the library ie. `PieItem`. Then the import on other projects would be `import PieChart, { PieItem } from 'react-minimal-pie-chart';`.

**Comment by manosim** [comment]
Hi @toomuchdesign! Left one comment but other than that, it looks good! 🎉

File: src/types/index.d.ts (hunk 1 above)
Comment (toomuchdesign) [reply src/types/index.d.ts]: I'll go with `PieChartData` and `PieChartExtendedData`, then.

**Pull request merged**